codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -156,13 +175,19 @@ public void sendFlowControlled(ChannelHandlerContext ctx, Http2Stream stream,
         }
         // Save the context. We'll use this later when we write pending bytes.
         this.ctx = ctx;
+        FlowState state;
```

Comment:
@buchgr - Thanks for opening a PR.  I am just using the default complier options set by our pom files.  I think `-Xlint:-options` is the only compiler arg.